Makefile

```diff
@@ -257,10 +257,15 @@ debian: sdist
 
 deb: deb-src
 	@for DIST in $(DEB_DIST) ; do \
+	    if test $${DIST} = trusty ; then \
+	        CRYPTO_BACKEND=pycrypto ; \
```

Comment:
we can do this via rules, just use dpkg-vendor and other toosl to figure out which dependency to include in file, only using env var as override.